google-cloud-logging/src/test/java/com/google/cloud/logging/BaseSystemTest.java

```diff
@@ -335,26 +336,27 @@ public void testListMetricsAsync() throws ExecutionException, InterruptedExcepti
 
   @Test
   public void testWriteAndListLogEntries() throws InterruptedException {
-    String logName = formatForTest("test-write-log-entries-log");
-    String filter =
-        "logName = projects/" + logging().getOptions().getProjectId() + "/logs/" + logName;
+    String logId = formatForTest("test-write-log-entries-log");
+    LoggingOptions loggingOptions = logging().getOptions();
+    LogName logName = LogName.create(loggingOptions.getProjectId(), logId);
     StringPayload firstPayload = StringPayload.of("stringPayload");
     LogEntry firstEntry = LogEntry.newBuilder(firstPayload)
         .addLabel("key1", "value1")
-        .setLogName(logName)
+        .setLogName(logId)
         .setHttpRequest(HttpRequest.newBuilder().setStatus(500).build())
         .setResource(MonitoredResource.newBuilder("global").build())
         .build();
     JsonPayload secondPayload =
         JsonPayload.of(ImmutableMap.<String, Object>of("jsonKey", "jsonValue"));
     LogEntry secondEntry = LogEntry.newBuilder(secondPayload)
         .addLabel("key2", "value2")
-        .setLogName(logName)
+        .setLogName(logId)
         .setOperation(Operation.of("operationId", "operationProducer"))
         .setResource(MonitoredResource.newBuilder("cloudsql_database").build())
         .build();
     logging().write(ImmutableList.of(firstEntry));
     logging().write(ImmutableList.of(secondEntry));
+    String filter = "logName = " + logName.toString();
```

Comment:
I scanned https://cloud.google.com/logging/docs/view/advanced_filters very quickly, and I think it might be a good idea to quote the log name inside the string because it has special characters (the `/`). Since you format the filter multiple times, it's probably worth making a helper method for it.